code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -347,6 +347,15 @@
 	name = "\improper MEUSOC .45"
 	desc = "Some serious drywall work, coming up!"
 */
+//hzdonut:Jesse Soemmer
+/obj/item/weapon/gun/projectile/revolver/fluff/jesselemat
+	name = "Modified LeMat"
+	desc = "The LeMat Revolver is a 9 shot revolver with a secondary firing barrel for loading shotgun shells. Uses .38-Special and 12g rounds depending on the barrel. This one appears to have had it's secondary barrel sealed off and looks to be in pristine condition. Either it's brand new, or its owner takes very good care of it."
+	icon_state = "lemat"
+	caliber = ".38"
+	max_shells = 9
+	ammo_type = /obj/item/ammo_casing/a38
+	projectile_type = /obj/item/projectile/bullet/pistol
```

Comment:
This line still needs to be deleted. It's dead code.

Reply:
If you look at the revolver code, you'll see that it's all .357, the revolver with .38 ammo is the detective revolver that has it's own set of special nonsense. If you're saying that deleting this won't cause me to end up with a revolver that shoots .357 ammo though i'll believe ya.

Reply:
`projectile_type = /obj/item/projectile/bullet/pistol` doesn't even belong in the .357 code. I should fix that.

`ammo_type = /obj/item/ammo_casing/a38` is necessary to decide what ammo spawns in the gun to start with.

Reply:
So I should re-add the ammo_type = /obj/item/ammo_casing/a38 code then? If I delete the line about /obj/item/projectile/bullet/pistol then it _should_ instead take the /obj/item/projectile/bullet/pistol/strong code from the revolver code which is equivalent to the Mateba, which I'm sure you guys don't want.

Reply:
That should not be in the original code, nor does it actually do anything to begin with. It should make no difference if it's removed and I should go remove it from Polaris.

Reply:
Ah, I see. Alright then, I'll leave things untouched.